target/linux/qualcommbe/patches-6.12/0361-net-pcs-Add-10GBASER-interface-mode-support-to-IPQ-U.patch

```diff
@@ -1,18 +1,20 @@
-From e770b36f0353fd11c4628360fe412acb7f02f346 Mon Sep 17 00:00:00 2001
+From 4a7db1e6ae3ca1ef42d7c95956f45d9233282089 Mon Sep 17 00:00:00 2001
 From: Lei Wei <quic_leiwei@quicinc.com>
 Date: Wed, 6 Mar 2024 17:40:52 +0800
-Subject: [PATCH] net: pcs: Add 10GBASER interface mode support to IPQ UNIPHY
- PCS driver
+Subject: [PATCH 1/8] net: pcs: Add 10GBASER interface mode support to IPQ
```

Comment:
I suggest to use the `--no-numbered` argument to `git format-patch` when generating patches. This way, the title stays the same so there is one less diff to worry about.